on_boot.d/99-lego-install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ ! -f /etc/cron.d/lego ]; then
+    /mnt/data/ssl/lego.sh renew
```

Comment:
So I was playing around with this too, and I was wondering if it doesn't make sense to add a sleep for like 180 seconds before the renew to avoid a scenario where a certificate is due for renewal, a reboot occurs, and the renew process interrupts some controller startup activities by restarting the controller(s) when this triggers.

What do you think?

Reply:
Yes of course, its a good idea.
Should I make a new PR, or will you do the modifications?